### ISSUE TITLE
Missing documentation for advanced config?

### ISSUE THREAD

**Issue by VictorChen** [PokemonGoF/PokemonGo-Bot#877]
```
max_steps : ??
mode : ??
walk : ??
debug : Let the default value here except if you are developper
test : Let the default value here except if you are developper
initial_transfer : Set this to 1 if you want to transfer pokemon
location_cache : ??
distance_unit : ??
item_filter : ??
evolve_all : Set to true to evolve pokemons if possible
```

https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/README.md#advance-configuration

**Comment by BartKoppelmans** [comment]
Will be fixed by #868 😉

**Comment by surfaace** [comment]
readme is updated
